Java Script/ex-21/index.js

```diff
@@ -0,0 +1,5 @@
+alert ("Enter two numbers to see if the first is greater than the second");
+
+let message = prompt('Enter the first number');
```

Comment:
when you are naming variables, make sure they are meaningful. It would make more sense to rename these variables as `firstNumber` and `secondNumber` 🙂 

Other than that great job @thanhnguyentanny 💯

Reply:
Thanks @chriztie !